src/scss/components/statistics.scss

```diff
@@ -0,0 +1,128 @@
+.statistics {
+  margin: 0;
```

Comment:
I'm not sure if these styles are doing anything. Can you double check if they are needed?

Reply:
You are right. I checked and changed it :) Thanks !